src/js/common/schemaform/review/ReviewPage.jsx

```diff
@@ -141,6 +159,8 @@ class ReviewPage extends React.Component {
               <ReviewCollapsibleChapter
                 key={chapter}
                 onEdit={this.handleEdit}
+                toggleChapter={() => this.handleToggleChapter(chapter)}
```

Comment:
Have had a lot of debate about naming props on stateless functional components i.e. should the name convey an intent (toggleChapter) or should it describe the action (toggleButtonClicked).

Reply:
I think we're all over the place with this. I think I generally prefer the latter.